### PR TITLE
add tool-tip and shortcut keyboard functionality to the swap handler

### PR DESCRIPTION
**Pull request by SantoshChapagai**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -298,6 +318,7 @@ export default function Home() {
             </div>
             <div>
             <IoMdSwap
+            title='Swap languages'
```
Comment (unellma) [inline pages/index.tsx:321]: Add text e.g. "Swap languages (Cmd + Shift + S)"

**Comment by amitkalwar** [comment]
I try in windows it's not working 
ctrl+shift+s 
Please check in mac it's ok

**Comment by unellma** [comment]
[UnelmaSupport](https://unelmasupport.com)

# Hello!

Thanks for getting in touch! This is an automatic email just to let you know that we've received your request and your ticket reference is #700.

One of our support agents will get back to you shortly. Please do not submit multiple tickets for the same request, as this will not result in a faster response time.

View Help Center: https://unelmasupport.com/hc

Regards,
UnelmaSupport

If you’re having trouble clicking the "View Help Center" button, copy and paste the URL below into your web browser: [https://unelmasupport.com/hc](https://unelmasupport.com/hc)

© 2024 UnelmaSupport. All rights reserved.

**Review by amitkalwar — APPROVED** [review]
working well

**Comment by unellma** [comment]
[UnelmaSupport](https://unelmasupport.com)

# Hello!

Thanks for getting in touch! This is an automatic email just to let you know that we've received your request and your ticket reference is #705.

One of our support agents will get back to you shortly. Please do not submit multiple tickets for the same request, as this will not result in a faster response time.

View Help Center: https://unelmasupport.com/hc

Regards,
UnelmaSupport

If you’re having trouble clicking the "View Help Center" button, copy and paste the URL below into your web browser: [https://unelmasupport.com/hc](https://unelmasupport.com/hc)

© 2024 UnelmaSupport. All rights reserved.

**Pull request merged**